diff --git a/cmake/modules/platform/arch/aarch64.cmake b/cmake/modules/platform/arch/aarch64.cmake
index 9f2617c1..dd6518a7 100644
--- a/cmake/modules/platform/arch/aarch64.cmake
+++ b/cmake/modules/platform/arch/aarch64.cmake
@@ -31,4 +31,9 @@ list(APPEND OMR_PLATFORM_DEFINITIONS
 set(TR_HOST_ARCH aarch64)
 set(TR_HOST_BITS 64)
 
+# we can only build jitbuilder for now and DDR is unsupported in crosscompile
+set(OMR_JITBUILDER ON CACHE BOOL "")
+set(OMR_COMPILER ON CACHE BOOL "")
+set(OMR_DDR OFF CACHE BOOL "")
+
 list(APPEND TR_COMPILE_DEFINITIONS TR_HOST_ARM64 TR_TARGET_ARM64 TR_HOST_64BIT TR_TARGET_64BIT BITVECTOR_64BIT)
